evaluators/text/BleuEvaluator/__init__.py

```diff
@@ -0,0 +1,62 @@
+from jina.executors.evaluators.text import BaseTextEvaluator
+
+class BleuEvaluator(BaseTextEvaluator):
+    """
+    :class:`BleuEvaluator`Bilingual Evaluation Understudy Score. 
+    Evaluates the generated sentence (actual) against a desired sentence. 
+    It will use the Bleu on NLTK package.
+    A perfect match will score 1.0 and a complete mismatch will score 0.0
+
+    The NLTK library can score n-gram individually or cummulative.
+    Here we use the cumulative as it is more precise.
+    https://machinelearningmastery.com/calculate-bleu-score-for-text-python/
+
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+
+    @staticmethod
+    def get_score(actual_list, desired_list, n_gram):
+        '''
+        Cumulative score is the calculation of individual n-grams
+        from 1 to n-order, and then weights them with the geometric mean
+        - desired_list, actual_list: are the sentences to be scored
```

Comment:
have a standard `docstring` with the proper `params`. I do not understand `It will check the biggest n_gram possible` if the `n_gram` is an input to the function.

Reply:
According to `bleu` these are the parameters: 

```
  :param references: reference sentences
    :type references: list(list(str))
    :param hypothesis: a hypothesis sentence
    :type hypothesis: list(str)
```